docs/cli/odigos_update-offsets.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: "odigos update-offsets"
+sidebarTitle: "odigos update-offsets"
+---
+## odigos update-offsets
```

Comment:
add a link from https://docs.odigos.io/instrumentations/golang/ebpf#about-go-offsets to this CLI command so users are aware how to fix the offset issue if they encounter it